compiler/rustc_target/src/spec/wasm32_unknown_emscripten.rs

```diff
@@ -19,10 +15,12 @@ pub fn target() -> Target {
         // functionality, and a .wasm file.
         exe_suffix: ".js".into(),
         linker: None,
+        pre_link_args,
```

Comment:
This `pre_link_args` is empty and will remove the ones defined in `wasm_base`, that seems unintentional ?

Reply:
Emscripten targets only use `LinkerFlavor::Em`, so the flags for for other flavors are indeed removed to satisfy the assert `assert_matches!(flavor, LinkerFlavor::Em)` in tests.

Reply:
I'll add a comment.

Reply:
I see, great, thanks for the clarification.